egs/wsj/s5/steps/data/data_lib.py

```diff
@@ -0,0 +1,23 @@
+import subprocess
```

Comment:
You could follow @jtrmal 's suggestion and rename this file. May be something like `data_dir_manipulation_lib.py`. 

Remove the libraries which have been commented out.